### DISCUSSION TITLE
Deeper nesting in Table of Contents / Page Navigation

### DISCUSSION THREAD

**Discussion by Genises** [colinwilson/lotusdocs#87] [Q&A] [answered]
Hello folks, 
is there a way to have the table of contents / page navigation show more than the first two markdown headers (e.g. ## and ###)? My documents consist of many subheadings, and choosing how many subheadings to show in the "on this page" section or previewing at least three instead of two sections would be very helpful in my use-case.

**Comment by illlustr** [comment, marked as answer]
in `hugo.toml' u can set
[markup]
    [markup.tableOfContents]
      endLevel = 'int'
      startLevel = 'int'
u can get nice example in https://github.com/colinwilson/lotusdocs.dev/

**Comment by Nadine2016** [comment]
thank you @illlustr that worked for me for the right-hand nav. not sure if @Genises also requires a deeper nav on the left as well? But the settings do not seem to change anything there.

**Comment by colinwilson** [comment]
Yep. As @illlustr mentions the below setting in `hugo.toml` affect the visible ToC levels.
```
[markup]
[markup.tableOfContents]
endLevel = 'int'
startLevel = 'int'
```
